fix cstyle error in src/replication_test.c #2003

divyanshkul opened the pull request:
Fixes [#2003](https://github.com/openebs/openebs/issues/2003)

divyanshkul committed:
fix cstyle issues 1

divyanshkul committed:
fix cstyle issues 2

divyanshkul commented:
Ping @ashishranjan738 @vishnuitta @payes

vishnuitta reviewed (commented):
There seems to be some new issues in cstyle check like #86, #329 lines of new file.
can you please run the './cstyle.pl <filename>' and provide the output?

divyanshkul committed:
fix cstyle issues 3

divyanshkul committed:
fix cstyle issues 4

divyanshkul committed:
fix cstyle issues 5

divyanshkul commented:
@vishnuitta  did not have any issues at #86, #329. Had some other ones, fixed them. Now all that is remaining are >80 characters in the code. I fixed most of them which included strings, like 
```
REPLICA_ERRLOG("Failed to send response"
" replica(%d)\n", ctrl_port);

```

but I don't know C so have no idea about fixing strings like
```
count = test_read_data(events[i].data.fd, (uint8_t *)data + recv_len, total_len - recv_len);

```

Rest all the cstyle issues are fixed.

divyanshkul commented:
@vishnuitta Travis build is passing, why is there an error with Better Code Hub (it isn't showing me errors as well, only admins can see)

divyanshkul commented:
Ping @vishnuitta

vishnuitta commented on src/replication_test.c:
```diff
@@ -75,22 +75,24 @@ init_mdlist(char *vol_name)
 	mdlist_fd = open(mdpath, O_CREAT|O_RDWR, 0666);
 	if (mdlist_fd < 0) {
-		REPLICA_ERRLOG("Failed to open metadata file %s err(%d)\n",
+		REPLICA_ERRLOG("Failed to open"
```
this seems to be unnecessary change. This is not part of cstyle errors of this file replication_test.c

divyanshkul replied:
It was showing me error on this line (when I ran it on my pc) @vishnuitta

vishnuitta replied:
can I know the errors that you saw?

divyanshkul replied:
Cstyle character>80 warning

divyanshkul replied:
Cstyle character>80 warning @vishnuitta

vishnuitta replied:
but, there are not more than 75 characters in that line. Can that be the case due to tabwidth in your editor?
Please set tabwidth as 8 in your editor.

vishnuitta reviewed (changes requested):
there are unnecessary changes. Fix only the errors shown in cstyle as per the issue#2003.

vishnuitta commented on src/replication_test.c:
```diff
@@ -323,11 +327,11 @@ send_mgmt_ack(int fd, zvol_op_code_t opcode, void *buf, char *replica_ip,
-					iovec[i].iov_len -= rc;
-					break;
-				} else {
+						iovec[i].iov_base
```
here, indentation went wrong.

divyanshkul committed:
fix cstyle issues 6

divyanshkul commented:
@vishnuitta sorry, there was no cstyle error in that line (my mistake :') ). Rest fixed that line and fixed the indentation error. :)

divyanshkul committed:
fix cstyle issues 7

divyanshkul commented:
@vishnuitta please review now :)

payes committed:
Merge branch 'replication' into fix-replication

payes commented:
@divyanshkul cstyle check still seems to fail.
Can you please re-run cstyle on the file and check.

divyanshkul commented:
@payes 
Now all that is remaining are >80 characters in the code. I fixed most of them which included strings, like 
```
REPLICA_ERRLOG("Failed to send response"
" replica(%d)\n", ctrl_port);

```

but I don't know C so have no idea about fixing strings like
```
count = test_read_data(events[i].data.fd, (uint8_t *)data + recv_len, total_len - recv_len);

```

Rest all the cstyle issues are fixed.
